packages/mdc-menu-surface/_mixins.scss

```diff
@@ -20,8 +20,53 @@
 // THE SOFTWARE.
 //
 
+@import "@material/feature-targeting/functions";
+@import "@material/feature-targeting/mixins";
 @import "@material/theme/mixins";
 @import "@material/shape/mixins";
+@import "@material/animation/variables";
+@import "@material/elevation/mixins";
+@import "@material/rtl/mixins";
+@import "./variables";
+
+
+//
+// Public
+//
+@mixin mdc-menu-surface($query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+  $feat-structure: mdc-feature-create-target($query, structure);
+  $feat-animation: mdc-feature-create-target($query, animation);
+
+  // postcss-bem-linter: define menu-surface
+  .mdc-menu-surface {
+    @include mdc-menu-surface-base_($query);
+
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-elevation(8);
+      @include mdc-menu-surface-fill-color(surface);
+      @include mdc-menu-surface-ink-color(on-surface);
+    }
+
+    @include mdc-feature-targets($feat-structure) {
+      @include mdc-rtl-reflexive-property(transform-origin, top left, top right);
+      @include mdc-menu-surface-shape-radius(medium);
+    }
+  }
+
+  @include mdc-feature-targets($feat-structure) {
+    .mdc-menu-surface--anchor {
+      position: relative;
+      overflow: visible;
+    }
+
+    .mdc-menu-surface--fixed {
+      position: fixed;
+    }
+  }
+
+  // postcss-bem-linter: end
+}
 
 @mixin mdc-menu-surface-ink-color($color) {
```

Comment:
Based on the discussion from the switch PR it sounds like we want to add `$query` support to all of the public mixins, that would mean these 3 too (and make sure to add them to the test file)